### ISSUE TITLE
npm install -g is omitted on generated Dockerfile

### ISSUE THREAD

**Issue by guiaramos** [rudxde/lerna-dockerize#200]
**Describe the bug**
on Dockerfile.base, I specified `RUN npm install -g @nestjs/cli` but this is line is being ignored on Dockerfile (generated)

**To Reproduce**
1. Go to Dockerfile.base and add the line `RUN npm install -g @nestjs/cli`
2. Run `npm run lerna-dockerize`
4. See  generated Dockerfile line `RUN npm install -g @nestjs/cli` is omitted

**Expected behavior**
It should add the line `RUN npm install -g @nestjs/cli` to the generated Dockerfile

**Screenshots**

![Screen Shot 2021-11-14 at 12 30 35 AM](https://user-images.githubusercontent.com/49703106/141649623-0f5b7ba5-1caa-476d-ab54-50c2134d3b86.png)

**Desktop (please complete the following information):**
 - OS: macOS Monterey Version 12.0.1

**Additional context**
```json
    "lerna": "4.0.0",
    "lerna-dockerize": "^0.8.2",
    "npm": ">= 7.24.0",
```

**Comment by rudxde** [comment]
i will look to fix this bug, but i can also suggest an workaround.
Instead of installing the cli global you can use npx to run the commands, or you install the cli as a dev-dependency and add a npm script. This would give you also better control over the installed version.

**Comment by guiaramos** [comment]
@rudxde thanks for always helping, when I get some free time I will try to submit a PR also.

**Comment by rudxde** [comment]
The issue should be solved in the next release (version 0.8.3)

**Comment by rudxde** [comment]
I will close the issue, for further problems just open a new issue.
